docs/contributing/API Review Process.md

```diff
@@ -0,0 +1,48 @@
+    * **Close as won't fix**. Similarly, if proposal is taking the product in a direction we simply don't want to go, the issue might also get closed. In that case, the problem isn't the proposed design but in the issue itself.
+
+5. **API gets reviewed**. The group conducting the review is called *RAR*, which stands for *roslyn api reviewers*. In the review, we'll take notes and provide feedback. After the review, we'll publish the notes in the [API Review repository](https://github.com/dotnet/apireviews) and at the end of the relevant issue. Multiple outcomes are possible:
+
+    * **Approved**. In this case the label `api-ready-for-review` is replaced
+    with `api-approved`.
+    * **Needs work**. In case we believe the proposal isn't ready yet, we'll
+    replace the label `api-ready-for-review` with `api-needs-work`.
+    * **Rejected**. In case we believe the proposal isn't a direction we want to go after, we simply write a comment and close the issue.
+
+## Review schedule
+
+ There are three methods to get an API review:
+
+* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/roslyn` and applying the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while. Progress of issues can be tracked via https://apireview.net/backlog/roslyn (PROTOTYPE: make this link actually work).
+* **Fast track**. If you need to bypass the backlog apply both `api-ready-for-review` and `blocking`. All blocking issues are looked at before we walk the backlog.
+* **Dedicated review**. This only applies to area owners. If an issue you are the area owner for needs an hour or longer, send an email to roslynapiowners and we book dedicated time. We also book dedicated time for language feature APIs: any APIs added as part of a new language feature need to have a dedicated review session. Rule of thumb: if the API proposal has more than a dozen APIs, the APIs have complex policy, or the APIs are part of a feature branch, then you need 60 min or more. When in doubt, send mail to roslynapiowners.
+
+The API review board currently meets biweekly.
+
+## Pull requests
+
+Pull requests against **dotnet/roslyn** shouldn't be submitted before getting approval. Also, we don't want to get work in progress (WIP) PR's. The reason being that we want to reduce the number pending PRs so that we can focus on the work the community expects we take action on.
```

Comment:
The file is already all about the API review, but making this explicit in this statement so that it's not misunderstood 

```suggestion
Pull requests against **dotnet/roslyn** that adds new public API shouldn't be submitted before getting approval. Also, we don't want to get work in progress (WIP) PR's. The reason being that we want to reduce the number pending PRs so that we can focus on the work the community expects we take action on.
```